figures/scripts/oncoprint-landscape.R

```diff
@@ -58,49 +58,41 @@ opt <- optparse::parse_args(opt_parser)
 lgat_png <- file.path(onco_dir, paste0(opt$lead_filename, "_lgat_goi_oncoprint.png"))
 embryonal_png <- file.path(onco_dir, paste0(opt$lead_filename, "_embryonal_goi_oncoprint.png"))
 hgat_png <- file.path(onco_dir, paste0(opt$lead_filename, "_hgat_goi_oncoprint.png"))
-ependymal_png <- file.path(onco_dir, paste0(opt$lead_filename, "_ependymal_goi_oncoprint.png"))
 other_cns_png <- file.path(onco_dir, paste0(opt$lead_filename, "_other_goi_oncoprint.png"))
 
 #### Assemble multipanel figure ------------------------------------------------
 
-oncoprint_figure <- multi_panel_figure(columns = 8,
-                                            rows = 2,
-                                            width = 1200,
-                                            height = 300)
+oncoprint_figure <- multi_panel_figure(columns = 4,
```

Comment:
One thing I noticed is that it almost looks like the titles for panels in the lower row are being cutoff ever so slightly during assembly. However, it kind of looks like that in the original PNGs, too. I am wondering if decreasing the `titleFontSize` or increasing the margin at the top a tiny bit might remedy that (in `02-plot-oncoprint.R`)

If we think the problem is instead assembly with `multi_panel_figure()` such that the white background of the top PNGs is responsible for the cutoff, another idea is to make the background of the PNGs transparent, which you can do with `magick` ([ref](https://stackoverflow.com/questions/50479875/how-to-remove-background-of-images-with-the-magick-package)), e.g., in this script do something like the following for each panel:

```r
img_png <- magick::read_image(<path to image>)
img_png <- magick::image_transparent(img_png, 'white')
```

I feel like it's more likely the first thing though in examining the PNGs by broad histology. I'll also note that if this takes more than ~1 hour, I think we've gone down a not super pressing rabbit hole. I also am aware that this can take awhile to run so I would certainly not run the whole module to find out, I'd start with a single broad histology!

Reply:
I also believe it is the first thing as I noticed this yesterday and went down a mini rabit hole trying to increase the margins but I was unable to successfully increase the top margins so I decreased the title size in the [most recent commit](https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1164/commits/b764e012db44f276a4f0abc644f22f6d77cd42f9) (and also slightly adjusted the height of the plots).